nuitka/distutils/bdist_nuitka.py

```diff
@@ -105,14 +105,27 @@ def _buildPackage(self, build_lib):
             "--output-dir=%s" % output_dir,
             "--recurse-dir=%s" % self.main_package,
             "--recurse-to=%s" % self.main_package,
-            "--recurse-not-to=*.tests",
             "--show-modules",
-            "--remove-output",
-            main_filename,
+            "--remove-output"
         ]
 
+        # Process any extra options from setuptools
+        if 'nuitka' in self.distribution.command_options:
+            for option, details in self.distribution.command_options['nuitka'].items():
+                option = "--" + option.lstrip('-')
+                source, value = details
+                if value is None:
+                    command.append(option)
+                elif hasattr(value, '__iter__'):
```

Comment:
Not sure if this ought to be: isinstance(value, collections.Iterable) and we need a string check first, there is something in nuitka.__past__  for that, as "basestring" is not available in Python3. Otherwise strings are iterables, and will fall into this.

Reply:
I haven't fixed this yet either sorry